src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -424,6 +482,38 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
         this.totalRelativeScore = this.calculateTotalScoreForTheCourse(RELATIVE_SCORE);
     }
 
+    calculateReachableScores() {
+        const quizzesTotalCurrentMaxScore = this.calculateScoreTypeForExerciseType(ExerciseType.QUIZ, REACHABLE_SCORE);
```

Comment:
Why do you need all those constants? As far as I can see you you just assign them below to reachable scores?